packages/terra-icon/tests/wdio/icon-spec.js

```diff
@@ -0,0 +1,38 @@
+/* global browser, Terra, before */
+describe('Icon', () => {
+  before(() => browser.setViewportSize(Terra.viewports('medium')[0]));
+  describe('Default', () => {
+    before(() => browser.url('/#/raw/tests/icon/icon-default'));
+
+    Terra.should.beAccessible();
+    Terra.should.matchScreenshot();
+  });
+
+  describe('Append', () => {
+    before(() => browser.url('/#/raw/tests/icon/icon-append-class'));
+
+    Terra.should.beAccessible();
+    Terra.should.matchScreenshot();
+  });
+
+  describe('Aria', () => {
```

Comment:
This test can be removed as it is covered by Jest snapshot test.

Reply:
https://github.com/cerner/terra-core/pull/1530/commits/8e1e43bda772216f1638a0621580584836218f73